ui/raidboss/browser_tts_engine.ts

```diff
@@ -56,7 +57,7 @@ export default class BrowserTTSEngine {
 
     // figure out what TTS engine type we need
     const speechLang = cactbotLangToSpeechLang[this.cactbotLang];
-    const voice = window.speechSynthesis.getVoices().find((voice) => voice.lang === speechLang);
+    const voice = window.speechSynthesis.getVoices().find((voice) => voice.lang.replaceAll('_','-') === speechLang);
```

Comment:
```suggestion
    const voice = window.speechSynthesis.getVoices().find((voice) => voice.lang.replaceAll('_', '-') === speechLang);
```

EDIT: sorry, nevermind.  This is correct, but lint will probably complain about the formatting.